tests/Doctrine/Tests_PHP74/Persistence/Mapping/RuntimeReflectionServiceTest.php

```diff
@@ -44,10 +54,43 @@ public function testGetTypedDefaultReflectionProperty() : void
         self::assertNotInstanceOf(TypedNoDefaultReflectionProperty::class, $reflProp);
     }
 
+    public function testGetNonTypedNoDefaultReflectionProperty() : void
+    {
+        $reflProp = $this->reflectionService->getAccessibleProperty(self::class, 'nonTypedNoDefaultProperty');
+        self::assertInstanceOf(ReflectionProperty::class, $reflProp);
+    }
+
+    public function testGetNonTypedDefaultReflectionProperty() : void
+    {
+        $reflProp = $this->reflectionService->getAccessibleProperty(self::class, 'nonTypedDefaultProperty');
+        self::assertInstanceOf(ReflectionProperty::class, $reflProp);
+        self::assertNotInstanceOf(TypedNoDefaultReflectionProperty::class, $reflProp);
+    }
+
     public function testGetTypedPublicNoDefaultPropertyWorksWithGetValue() : void
```

Comment:
This test got changed by your PR. If the old test using `RuntimePublicReflectionProperty` would fail with your changes, you might have implemented a BC-break here and 1.3.x wouldn't be the appropriate target branch.

Reply:
The old test fails with the fix - but wouldn't the old behaviour be considered a bug?

Reply:
A bugfix that introduces BC break could only be merged into a major release. The fix should also be reviewed by other maintainers too.

Reply:
This fix has been hold off for too long... as it is it we may not even have it for version 3.0... Wouldn't it be possible to preserve BC? e.g. use a new class that extends RuntimePublicReflectionProperty but use the same code as TypedNoDefaultReflectionProperty?

Reply:
Well my question was primarily to @SenseException (and other maintainers), but I was thinking to something like that yes 🙂 Thanks, now let's wait for new reviews 😉